src/site/_filters/csp-hash.js

```diff
@@ -2,11 +2,8 @@ const {sha256base64} = require('../_data/lib/hash');
 const hashList = new Set();
 
 function cspHash(raw) {
-  if (process.env.ELEVENTY_ENV === 'prod') {
-    const hash = `'sha256-${sha256base64(raw)}'`;
-    hashList.add(hash);
-  }
-  return raw;
```

Comment:
This return value is necessary no, since we are chaining filters together.

Reply:
oh! yes.